microvmConfigurations/netvm/default.nix

```diff
@@ -40,24 +41,26 @@ nixpkgs.lib.nixosSystem {
 
       networking.nat = {
         enable = true;
-        internalInterfaces = ["enp0s4"];
+        internalInterfaces = ["ethint0"];
       };
 
-      # TODO: Set the interface name to something pre-defined.
-      # Setting the name manually with ip link set <iface> name <newname>
-      # works. The following breaks things for some reason:
-      #
       # Set internal network's interface name to ethint0
-      # systemd.network.links."10-ethint0" = {
-      #   matchConfig.PermanentMACAddress = "02:00:00:01:01:01";
-      #   linkConfig.Name = "ethint0";
-      # };
+      systemd.network.links."10-ethint0" = {
```

Comment:
I actually want to test this before merge, because I had so many problems with this before, and I was suspecting they are bugs in the systemd-networkd, but didn't have time to dig enough to generate any bug report

Reply:
if hyphen is problematic with interface/systemd naming - `eth0debug`

Reply:
Should we create separate virtual bridge for debugging purposes, present only on debug builds?

Reply:
The idea is this virtual bridge is actually supposed to be used for sharing network from NetVM to other VMs

Reply:
(Also I think there's some limit how long the interface name can be)

Reply:
Yes, ethint0 is for sharing the network to other VMs from NetVM.

Regarding the debugging ssh access: I just wonder what would be the least messy solution when one more VM (IDS-VM) between NetVM and other VMs.